study.md

```diff
@@ -27,17 +27,21 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+its not the html and and its not Javascript. html can viewed as the base/foundation model of DOM and
+Javascript can be viewed as the manipulator of DOM.
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+Yes, dev tools are the visual representation of DOM, and it will look like your html file in most cases.
```

Comment:
The DOM is an interface that represents HTML documents in a tree structure.  What you see in your web browser's developer tools is a visual representation of the browser's implementation of that interface, but not the DOM itself.